clang/test/Driver/range.c

```diff
@@ -44,3 +47,4 @@
 // CHECK-NOT: -complex-range=fortran
 // WARN1: warning: overriding '-fcx-limited-range' option with '-fcx-fortran-rules' [-Woverriding-option]
 // WARN2: warning: overriding '-fcx-fortran-rules' option with '-fcx-limited-range' [-Woverriding-option]
+// range-no-diagnostics
```

Comment:
CC @MaskRay is this a reasonable way to test or should we be using `CHECK-NOT` and not using `-verify` because the diagnostics come from the driver?

Reply:
`-verify` does not work for driver errors/warnings. You can add `-Werror`.  In case of errors, `clang -###` will exit with code 1. Since lit requires that commands exit with code 0, `-### -Werror` alone serves as a test.

```
% clang -### -c -fcx-fortran-rules a.c -Werror
...
clang: error: overriding '' option with '-fcx-fortran-rules' [-Werror,-Woverriding-option]
...
```

Reply:
@MaskRay Thanks. Let me know if the test is correct now.

Reply:
Delete the unused `// range-no-diagnostics`